src/block/block_compact.c

```diff
@@ -89,6 +90,8 @@ __wt_block_compact_get_progress_stats(
     WT_STAT_DATA_SET(session, btree_compact_pages_rewritten, block->compact_pages_rewritten);
     WT_STAT_DATA_SET(
       session, btree_compact_pages_rewritten_expected, block->compact_pages_rewritten_expected);
+    WT_STAT_DATA_SET(
```

Comment:
Can we A->Z this list please?

Reply:
https://github.com/wiredtiger/wiredtiger/pull/10354/commits/bcb2bf27f40105d04ff087f9ccea66d81ee108cf